CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+## [Unreleased]
```

Comment:
In this section we can track changes on master since the last version release (so currently empty).  Ideally when making some relevant change, it should directly be listed there in the same PR.  If maintained properly, this makes updating the changelog for a new release easy, as we only need to rename the section accordingly.

Reply:
to make sure I understood correctly: all updates of the code on the master branch does not imply an update of the version number ?

Reply:
Yes, I think we don't need to update the version number for every change but only when we add a new version tag, e.g. if we want to pin versions for a publication or something like the challenge.

So I think the only significant difference to our current practice would be to have a bit more structured version tags, instead of tags like "real_robot_challenge_*" or "o80_joss".

Does that make sense to you?